src/lintlist/mod.rs

```diff
@@ -1525,6 +1526,13 @@ pub const ALL_LINTS: [Lint; 304] = [
         deprecation: None,
         module: "redundant_pattern_matching",
     },
+    Lint {
+        name: "redundant_static_lifetimes",
+        group: "style",
+        desc: "Using explicit `\'static` lifetime for constants or statics when elision rules would allow omitting them.",
```

Comment:
Is the backward slash \ necessary?

Reply:
No, but this Regex:
https://github.com/rust-lang/rust-clippy/blob/be5d17feb20534d503c49722beecf5501d5d9c3a/clippy_dev/src/lib.rs#L165-L167

probably adds it, while parsing the description of the lint. This could be handled here

https://github.com/rust-lang/rust-clippy/blob/be5d17feb20534d503c49722beecf5501d5d9c3a/clippy_dev/src/lib.rs#L49

just like `\"` is. 🤔